core/Plugin/ControllerAdmin.php

```diff
@@ -365,8 +366,8 @@ protected static function getPiwikVersion()
         return "Matomo " . Version::VERSION;
     }
 
-    private static function isPhpVersionAtLeast55()
+    private static function isPhpVersionAtLeast71()
```

Comment:
What do you think about renaming that method to `isMinimalSupportedPhpVersion()` or something along this line? There will be no need to rename it in the future.